src/Servers/Kestrel/Core/src/Internal/Http/Http1ChunkedEncodingMessageBody.cs

```diff
@@ -87,6 +87,7 @@ public override async ValueTask<ReadResult> ReadAsync(CancellationToken cancella
         public override void Complete(Exception exception)
         {
             _requestBodyPipe.Reader.Complete(exception);
+            _context.ReportApplicationError(exception);
```

Comment:
This will produce an application error log even if `exception` is null.

Reply:
I changed ReportApplicationError to return if exception is null.